pkgs/applications/audio/master_me/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, pkg-config
+, python3, Cocoa }:
+
+stdenv.mkDerivation rec {
+  pname = "master_me";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "trummerschlunk";
+    repo = "master_me";
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "sha256-FG3X1dOF9KRHHSnd5/zP+GrYCB2O0y+tnI5/l9tNhyE=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libGL libXext libXrandr ]
+                ++ lib.optionals stdenv.isDarwin [ Cocoa ]
+                ++ lib.optionals stdenv.isLinux [ libX11 ];
```

```suggestion
  buildInputs = [ libGL ]
    ++ lib.optionals stdenv.isDarwin [ Cocoa ]
    ++ lib.optionals stdenv.isLinux [ libX11 libXext libXrandr ];
```
those should be x11, too

---

Not sure what you mean by "those should be x11, too", but I've applied everything else.